components/disputes/DisputeCard.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <BorderCard
+    color="green"
+    class="mt-5"
+  >
+    <div>{{ dispute.name }}</div>
+    <div>STATUS: {{ dispute.status }}</div>
+    <div>RODADAS: {{ dispute.rounds }}</div>
+    <div>{{ dispute }}</div>
+  </BorderCard>
+</template>
+
+<script>
+import BorderCard from '@/components/shared/BorderCard'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rafaelmbcosta/league-nuxt/pullRequest?prid=7487498)
```suggestion
import BorderCard from "@/components/shared/BorderCard"
```